### PR TITLE
signal-desktop (aarch64) : 6.40.0 -> 6.42.0

### PR DESCRIPTION
**Pull request by Arjun31415**
## Description of changes

signal-desktop (aarch64) : 6.40.0 -> 6.42.0
signal-desktop-beta: 6.40.0-beta.2 -> 6.53.0-beta.1

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by teutat3s — COMMENTED** [review]
Thanks, I missed this in https://github.com/NixOS/nixpkgs/pull/274426. Just one nit: could you add a (aarch64) to the commit message?

**Comment by Arjun31415** [comment]
sure

**Review by teutat3s — COMMENTED** [review]
One more nit, there's a typo in the commit message:
> signal-desktop-beta: 6.40.0-beta.2 -> 6.53.0-beta.1

-> `signal-desktop-beta: 6.40.0-beta.2 -> 6.43.0-beta.1`

File: pkgs/applications/networking/instant-messengers/signal-desktop/signal-desktop-aarch64.nix [hunk 1]
```diff
@@ -2,7 +2,7 @@
 callPackage ./generic.nix { } rec {
   pname = "signal-desktop";
   dir = "Signal";
-  version = "6.40.0";
+  version = "6.42.0";
   url = "https://github.com/0mniteck/Signal-Desktop-Mobian/raw/${version}/builds/release/signal-desktop_${version}_arm64.deb";
   hash = "sha256-3Pi0c+CGcJR1M4ll51m+B5PmGIcIjjlc0qa9b8rkMeU=";
```
Comment (eclairevoyant) [inline pkgs/applications/networking/instant-messengers/signal-desktop/signal-desktop-aarch64.nix:7]: please fix the hash

**Review by eclairevoyant — CHANGES_REQUESTED** [review]

**Comment by Mic92** [comment]
Is this one not obsolete since: https://github.com/NixOS/nixpkgs/pull/274426 ?

**Comment by eclairevoyant** [comment]
No, there's 3 packages, only one of which was updated in the previous PR. The PR title is wrong

**Comment by Arjun31415** [comment]
Running the update script for updating the hash of aarch64 fails because I do not have aarch64 system. However I don't think an aarch64 system is required to update only the hashes

**Comment by eclairevoyant** [comment]
Easiest way is to set up binfmt, or copy the value from my PR

**Review by eclairevoyant — APPROVED** [review]

**Comment by eclairevoyant** [comment]
Just make sure to update the commit message to the right version number, and it's fine otherwise

**Pull request merged**